### DISCUSSION TITLE
充值问题……

### DISCUSSION THREAD

**Discussion by YuChen-Ethan** [ZuodaoTech/everyone-can-use-english#1119] [Q&A]
老师好，想咨询一下充值问题：
充值两美元为什么只能支付3.37新加坡元？3.37SGD=18.2CNY，$2=￥14.1，需要多付大约4元人民币。
![image](https://github.com/user-attachments/assets/9925b63b-41cc-4292-95dc-38643dfa1c09)

**Comment by an-lee** [comment]
法币充值有渠道费，除了固定比率抽成，还有每笔固定的费用。充 2 刀的手续费太高了，10 刀的话，手续费平均下来会便宜点。

有条件的话可以考虑用虚拟货币充值渠道，没有额外的渠道费。
